modules/core/src/main/scala/com/azavea/stac4s/extensions/layer/LayerItemExtension.scala

```diff
@@ -0,0 +1,25 @@
+package com.azavea.stac4s.extensions.layer
+
+import com.azavea.stac4s.extensions.ItemExtension
+
+import cats.Eq
+import io.circe.{Decoder, Encoder}
+import io.circe.syntax._
+
+case class LayerItemExtension(ids: List[String])
```

Comment:
I think this should be `NonEmptyString`

Reply:
...should it also be a `NonEmptyList`? I'm not sure about the semantics of what an empty list would mean. Don't do filtering? @pomadchin

Reply:
@jisantuc technically if `layer:ids` fields is defined it can not be empty... if it is empty or missing it means that this item does not belong to the layer. 

¯\_(ツ)_/¯ did I answer your question?